source/tyr/tests/integration/migrate_poi_to_osm_test.py

```diff
@@ -0,0 +1,130 @@
+from tests.check_utils import api_get, api_delete
```

Comment:
license

Reply:
Done. Only 1 file in the directory contains license header. Should be added in every files if needed